vulcanai2/models/utils.py

```diff
@@ -174,34 +178,82 @@ def hook(module, input, output):
 
     return summary
 
+
 def print_model_structure(network, input_size=None):
-    """
-    Print the entire model structure.
-    
-    """
+    """Print the entire model structure."""
     shapes = network_summary(network)
     for k, v in shapes.items():
         print('{}:'.format(k))
         if isinstance(v, odict):
             for k2, v2 in v.items():
                 print('\t {}: {}'.format(k2, v2))
 
+
+def selu_weight_init_(tensor, mean=0.0):
+    """
+    SELU layer weight initialization function.
+
+    Function assigned to variable that will be called within
+    _init_weights function to assign weights for selu.
+
+    Parameters
+    ----------
+    tensor :  torch.tensor
+        Weight tensor to be adjusted
+    mean : float
+        Mean value for the normal distribution
+
+    Returns
+    -------
+    torch.tensor
+        weight tensor with normal distribution
+
+    """
+    with torch.no_grad():
+        fan_in, _ = nn.init._calculate_fan_in_and_fan_out(tensor)
+        std = math.sqrt(1. / fan_in)
+        return nn.init.normal_(tensor, mean, std)
```

Comment:
This function seems identical to:
`nn.init.kaiming_normal_(mode="fan_in", nonlinearity="linear")`

Reply:
Hi @jcowles that's a great find! If you would like to submit a PR to update this, that would be fantastic.

cc @nirtiac